dbt_coves/tasks/fix.py

```diff
@@ -27,7 +28,16 @@ def register_parser(cls, sub_parsers, base_subparser):
         return subparser
 
     def run(self) -> int:
-        for source_path in self.config.source_paths:
-            console.print(f"Trying to auto-fix linting errors in [u]{source_path}[/u]...\n")
-            fix(source_path)
+        objs_to_fix = list()
+        if "source_paths" in self.config.__dict__:
```

Comment:
This is not very "pythonic".
You should do 
`objs_to_fix = self.config.get("source_paths", self.config.get("model_paths"))
if not objs_to_fix:
  raise...`

Reply:
Changed as requested:

```
objs_to_fix = self.config.get("source_paths", self.config.get("model_paths"))
        if not objs_to_fix:
            raise DbtCovesException(
                "Could not find [u]source_paths[/u] or [u]model_paths[/u] in [u]dbt_project.yml[/u] file"
            )```